moped-api/claims.py

```diff
@@ -1,5 +1,6 @@
 import json
 from flask_cognito import _request_ctx_stack, current_cognito_jwt
+from functools import wraps
```

Comment:
This is cool! I learned something new today! :)

Reply:
Same here! Planning on sharing how I tripped over this in dev sync this week. 👍